rollup.config.js

```diff
@@ -52,37 +52,6 @@ export default [
       // Allow bundling cjs modules (unlike webpack, rollup doesn't understand cjs)
       commonjs(),
 
-      // Resolve source maps to the original source
-      sourcemaps(),
-    ],
-  },
-  {
-    input: "dist/lib/cli.js",
-    output: [
-      {
-        file: "bin/cli.js",
-        format: "cjs",
-        sourcemap: true,
-        banner: "#!/usr/bin/env node\n",
-      },
-    ],
-    // Indicate here external modules you don't wanna include in your bundle (i.e.: 'lodash')
-    external: ["fs", "path", "util"],
-    watch: {
-      include: "src/**",
-    },
-    plugins: [
-      // Allow json resolution
-      json(),
-      // Allow node_modules resolution, so you can use 'external' to control
-      // which external modules to include in the bundle
-      // https://github.com/rollup/rollup-plugin-node-resolve#usage
-      resolve(),
-      // Compile TypeScript files
-      // typescript(),
-      // Allow bundling cjs modules (unlike webpack, rollup doesn't understand cjs)
-      commonjs(),
-
```

Comment:
Without this the cli isn't built. I think we need this, but name it `bin/catalog-converter.js` (or `bin/catalog-converter`)